react-native.config.js

```diff
@@ -11,8 +11,14 @@
 const ios = require('@react-native-community/cli-platform-ios');
 const android = require('@react-native-community/cli-platform-android');
 
+// Remove commands so that react-native-macos can coexist with react-native in repos that depend on both.
+const path = require('path');
+const isReactNativeMacOS = path.basename(__dirname) === 'react-native-macos';
+const iosCommands = isReactNativeMacOS ? [] : ios.commands;
+const androidCommands = isReactNativeMacOS ? [] : android.commands;
+
 module.exports = {
-  commands: [...ios.commands, ...android.commands],
+  commands: [...iosCommands, ...androidCommands],
```

Comment:
Why don't we need a ...macCommands here too?

Reply:
We will, but mac commands don't exist yet.   We have to create a `@react-native-community/cli-platform-macos` package first.

Reply:
FYI, the commands can be in this repo.  No need to make that a separate repo.  Windows implements its commands within react-native-windows.

Reply:
The only reason the android and ios ones are separate is so that the community can handle the PRs etc without waiting on facebook, since facebook doesn't use the cli.